### DISCUSSION TITLE
How to troubleshoot timeouts?

### DISCUSSION THREAD

**Discussion by bjorn-ali-goransson** [gotenberg/gotenberg#1074] [Q&A]
Hello there and thanks for your work 

We are getting a web socket timeout reached error sometimes after 20+s.

We do not want to increase this timeout. The question is though why is it taking such a long time? 

I am suspecting that some network resource that is specified in the HTML is not responsive but we don't know. 

How can we check this easiest?

**Comment by gulien** [comment]
Hey @bjorn-ali-goransson 👋 

You may try with [failOnResourceHttpStatusCodes](https://gotenberg.dev/docs/routes#invalid-http-status-codes-chromium) and [failOnResourceLoadingFailed](https://gotenberg.dev/docs/routes#network-errors-chromium).
